props/saber_BC_buttons.h

```diff
@@ -1288,24 +1283,6 @@ struct BCVolumeMode : public SPEC::SteppedMode {
     SPEC::SteppedMode::exit();
   }
 
-  bool Parse(const char *cmd, const char* arg) override {
```

Comment:
Why did this go away?

Reply:
Okay I'm not at my computer right now, but this doesn't belong in the mode here, so it's just been moved down into the saber BC buttons class. However, I thought it was appropriate in the other PR but your comment there seems like no. I just want to move it.

Reply:
> Why did this go away?

Added in #741 instead.

Reply:
ugh. I got mixed up. 
Ok #741 is correct now (moving the code block).
Restored here so it's no longer a diff.